packages/plugins/typescript/nhost/package.json

```
@@ -0,0 +1,56 @@
+{
+  "name": "@graphql-codegen/typescript-nhost",
+  "version": "0.0.1",
```

```suggestion
  "version": "0.0.0",
```
We need to let changesets to make the release. So let's create a `patch` changeset using `yarn changeset` on the root

---

done